pkg/scaffold/api.go

```diff
@@ -86,90 +83,46 @@ func (s *apiScaffolder) newUniverse() *model.Universe {
 	)
 }
 
-func (s *apiScaffolder) scaffoldV1() error {
-	if s.doResource {
-		if err := machinery.NewScaffold().Execute(
-			s.newUniverse(),
-			&crdv1.Register{},
-			&crdv1.Types{},
-			&crdv1.VersionSuiteTest{},
-			&crdv1.TypesTest{},
-			&crdv1.Doc{},
-			&crdv1.Group{},
-			&crdv1.AddToScheme{},
-			&crdv1.CRDSample{},
-		); err != nil {
-			return fmt.Errorf("error scaffolding APIs: %v", err)
-		}
-	} else {
-		// disable generation of example reconcile body if not scaffolding resource
-		// because this could result in a fork-bomb of k8s resources where watching a
-		// deployment, replicaset etc. results in generating deployment which
-		// end up generating replicaset, pod etc recursively.
-		s.resource.CreateExampleReconcileBody = false
-	}
-
-	if s.doController {
-		if err := machinery.NewScaffold().Execute(
-			s.newUniverse(),
-			&controllerv1.Controller{},
-			&controllerv1.AddController{},
-			&controllerv1.Test{},
-			&controllerv1.SuiteTest{},
-		); err != nil {
-			return fmt.Errorf("error scaffolding controller: %v", err)
-		}
-	}
-
-	return nil
-}
-
 func (s *apiScaffolder) scaffoldV2() error {
```

Comment:
If we're removing `v2` from scaffold import paths, this should be
```suggestion
func (s *apiScaffolder) scaffold() error {
```

Ditto for other method/function names.

Reply:
good catcher